backend/src/migrations/20240815120000_update_email_length.ts

```diff
@@ -0,0 +1,18 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+const COLUMN_NAME = 'email';
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(TABLE_NAME, (table) => {
+        table.string(COLUMN_NAME, 320).alter();
```

Comment:
is not better to make another const for length? also, the table and column name shouldn't have the actual name ?

Reply:
Great suggestion about the 320 as a constant. The column name is defined as a const at the beginning of the file